pkg/proxy/replica.go

```diff
@@ -30,7 +31,30 @@ func (p *Proxy) ReplicaList(ctx context.Context, req *rpc.ProxyEngineRequest) (r
 		ReplicaList: &eptypes.ReplicaListReply{},
 	}
```

Comment:
NIT: there is no need to init `resp` at the beginning.

Reply:
Fixed.